.github/actions/prepare-runner/action.yml

```
@@ -0,0 +1,17 @@
+name: 'Setup runner'
+description: 'It setups pnpm, node and installs dependencies'
+runs:
+  using: composite
+  steps:
+    - name: Install pnpm
+      uses: pnpm/action-setup@v2
```

```suggestion
      uses: pnpm/action-setup@v4
```

Let’s update them if we change it

---

Updated